File: frontend/apps/admin/features/fa-im-pages/types/friend/ImFriendGroup.ts
// types/friend/ImFriendGroup.ts
export interface ImFriendGroup {
  id: number;              // 主键ID
  userId: number;          // 用户ID
  groupName: string;       // 分组名称
  sort?: number;           // 排序
  crtTime?: string;        // 创建时间
  updTime?: string;        // 更新时间
}

File: fa-im/src/main/java/com/faber/api/im/friend/entity/ImFriendGroup.java
package com.faber.api.im.friend.entity;

import com.faber.core.bean.BaseDelEntity;
import com.baomidou.mybatisplus.annotation.TableName;
import com.baomidou.mybatisplus.annotation.TableField;
import lombok.Data;
import lombok.EqualsAndHashCode;

/**
 * 好友分组实体类
 *
 * @author faber
 */
@Data
@EqualsAndHashCode(callSuper = false)
@TableName("im_friend_group")
public class ImFriendGroup extends BaseDelEntity {
    /**
     * 用户ID
     */
    @TableField("user_id")
    private Long userId;

    /**
     * 分组名称
     */
    @TableField("group_name")
    private String groupName;

    /**
     * 排序
     */
    @TableField("sort")
    private Integer sort;
}